### PR TITLE
Vlad turcan -  done with the MVP

### PR DESCRIPTION
**Pull request by justman00**
finished the mvp

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -3,144 +3,348 @@
...
+  border-bottom: 1px dashed black;
+}
+
+.card img {
+  width: 100%;
+}
+
+.last-part {
+  margin-top: 5%;
+  margin-bottom: 5%;
```
Comment (PhilHart23) [inline css/index.css:239]: Nice job making everything percent based

File: index.html [hunk 2]
```diff
@@ -1,75 +1,105 @@
...
+      <footer>
+        <nav>
+          <a href="#">Home</a> <a href="#">About</a> <a href="#">Products</a>
+          <a href="#">Blog</a> <a href="#">Contact</a>
+        </nav>
+      </footer>
+    </div>
+    <!-- container -->
+  </body>
+</html>
```
Comment (PhilHart23) [inline index.html:105]: Good job on the sprint!

File: index.html [hunk 3]
```diff
@@ -1,75 +1,105 @@
...
\ No newline at end of file
+      <footer>
+        <nav>
+          <a href="#">Home</a> <a href="#">About</a> <a href="#">Products</a>
+          <a href="#">Blog</a> <a href="#">Contact</a>
+        </nav>
+      </footer>
+    </div>
+    <!-- container -->
+  </body>
```
Comment (PhilHart23) [inline index.html:104]: Did you gorget to answer the questions?

**Comment by justman00** [comment]
Yes haha. Completely went out of my mind. The questions are quite simple anyway

Von meinem iPhone gesendet

**Pull request merged**